server/service/scenarios/db.js

```diff
@@ -336,6 +336,25 @@ async function getScenariosSlice(direction, offset, limit) {
   return scenarios;
 }
 
+async function getRecentScenarios(user, orderBy = 'updated_at', limit = 4) {
+  const results = await query(`
+    SELECT id
+    FROM scenario
+    WHERE author_id = ${user.id}
```

Comment:
Using `author_id` is "deprecated" in the sense that it shouldn't be used for anything new anymore, since scenarios can have _n_ collaborators who are all considered "authors" and one "owner". You'll want to take a look at `getScenarioUsers`.